cloud_governance/policy/ibm/cost_billing_reports.py

```diff
@@ -28,7 +28,7 @@ def __init__(self):
     def prepare_es_data(self, usage_cost: float, next_invoice: float):
         """This method prepares the data to upload to the es"""
         date = datetime.datetime.now()
-        start_date = f'{date.year}-{date.month}-01'
+        start_date = f'{date.year}-{date.strftime("%m")}-01'
```

Comment:
Why we need 01, I think that month and year is enough, WDYT ?

Reply:
We need to select always start_day of the month. So I included the 01